pkg/proto/core/v1/util.go

```diff
@@ -52,54 +42,60 @@ func CoreRelationTuples(tuples []*v0.RelationTuple) []*RelationTuple {
 	return coreTuples
 }
 
+// CoreRelationTupleUpdate converts the input to a core CoreRelationTupleUpdate.
 func CoreRelationTupleUpdate(update *v0.RelationTupleUpdate) *RelationTupleUpdate {
 	coreUpdate := RelationTupleUpdate{}
 	bytes, err := proto.Marshal(update)
-	defer panicOnError(err)
+	panicOnError(err)
 	err = proto.Unmarshal(bytes, &coreUpdate)
-	defer panicOnError(err)
+	panicOnError(err)
 	return &coreUpdate
 }
 
+// CoreObjectAndRelation converts the input to a core CoreObjectAndRelation.
 func CoreObjectAndRelation(onr *v0.ObjectAndRelation) *ObjectAndRelation {
 	coreOnr := ObjectAndRelation{}
 	bytes, err := proto.Marshal(onr)
-	defer panicOnError(err)
+	panicOnError(err)
 	err = proto.Unmarshal(bytes, &coreOnr)
-	defer panicOnError(err)
+	panicOnError(err)
 	return &coreOnr
 }
 
+// CoreZookie converts the input to a core Zookie.
 func CoreZookie(zookie *v0.Zookie) *Zookie {
 	if zookie == nil {
 		return nil
 	}
 	coreZookie := Zookie{}
 	bytes, err := proto.Marshal(zookie)
-	defer panicOnError(err)
+	panicOnError(err)
 	err = proto.Unmarshal(bytes, &coreZookie)
-	defer panicOnError(err)
+	panicOnError(err)
 	return &coreZookie
 }
 
+// CoreRelationReference converts the input to a core CoreRelationReference.
 func CoreRelationReference(ref *v0.RelationReference) *RelationReference {
 	coreRef := RelationReference{}
 	bytes, err := proto.Marshal(ref)
-	defer panicOnError(err)
+	panicOnError(err)
 	err = proto.Unmarshal(bytes, &coreRef)
-	defer panicOnError(err)
+	panicOnError(err)
 	return &coreRef
 }
 
+// CoreNamespaceDefinition converts the input to a core NamespaceDefinition.
 func CoreNamespaceDefinition(def *v0.NamespaceDefinition) *NamespaceDefinition {
```

Comment:
Add `To` in front of all of these

Reply:
Those should've been renamed in the last push but I'll double check